packages/blade/src/components/Tooltip/_decisions/decisions.md

```diff
@@ -0,0 +1,221 @@
+# Tooltip API Decisions <!-- omit in toc -->
+
+The tooltip typically provides additional context about the element or its function. A tooltip is always triggered by a `mouse hover` on desktop and `on tap` on mobile.
+
+<img src="./tooltip-thumbnail.png" width="380" alt="" />
+
+- [Design](#design)
+- [`Tooltip` API](#tooltip-api)
+  - [Examples:](#examples)
+    - [Usage](#usage)
+  - [Working with non-interactive triggers](#working-with-non-interactive-triggers)
+  - [Custom Triggers](#custom-triggers)
+- [Library](#library)
+  - [Bundle Size](#bundle-size)
+- [Motion](#motion)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+- [References](#references)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=40540-559304&t=tmTrf3xJU6oj59fM-0) to all variants of the Tooltip component
+
+## `Tooltip` API
+
+| Prop         | Type                                                                     | Default     | Description                                                                                                                             | Required |
+| ------------ | ------------------------------------------------------------------------ | ----------- | --------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| content      | `string`                                                                 | `undefined` | Content of the tooltip                                                                                                                  | ✅       |
+| children     | `React.ReactNode`                                                        | `undefined` | Trigger component for tooltip, Accepts any interactive element or icons                                                                 | ✅       |
+| placement    | `top, top-start, top-end, left, right, bottom, bottom-start, bottom-end` | `top`       | Placement of tooltip, the tooltip avoid collision with the edge of the screen and flip to the oppisite side even when placement is set. |          |
+| onOpenChange | `({ isOpen }) => void`                                                   | `undefined` | Called when tooltip isOpen state is changed, this can be used to detect when tooltip opens or closed                                    |          |
```

Comment:
Does `onOpen` and `onClose` sound better?

Reply:
Yeah discussed this earlier also. 
**[Check the discussion here](https://github.com/razorpay/blade/pull/1216#discussion_r1212981863)**

We will go with 1 handler only since we are doing the same in few other components as well, Accordion, Collapse etc.

Reply:
i dont think this stops anyone from writing inline handlers here as well, but if we are following the same in other components its fine